api/src/main/java/org/apache/cloudstack/api/response/TemplateResponse.java

```diff
@@ -189,18 +191,28 @@
     @Param(description = "KVM Only: true if template is directly downloaded to Primary Storage bypassing Secondary Storage")
     private Boolean directDownload;
 
+    @SerializedName(ApiConstants.DEPLOY_AS_IS)
+    @Param(description = "VMware only: true if template is deployed without orchestrating disks and networks but \"as-is\" defined in the template.")
+    private Boolean deployAsIs;
+
     @SerializedName("parenttemplateid")
     @Param(description = "if Datadisk template, then id of the root disk template this template belongs to")
+    @Deprecated(since = "4.15")
     private String parentTemplateId;
 
     @SerializedName("childtemplates")
     @Param(description = "if root disk template, then ids of the datas disk templates this template owns")
+    @Deprecated(since = "4.15")
     private Set<ChildTemplateResponse> childTemplates;
```

Comment:
@nvazquez did we get rid of child-template design from old multi-disk ova code? Can this be removed?

Reply:
Not removed, but unused when templates are registered "as-is"